src/elements/content-sidebar/activity-feed/activity-feed/ActivityItem.tsx

```diff
@@ -6,11 +6,24 @@ type Props = {
     children: React.ReactNode;
     className?: string;
     isFocused?: boolean;
+    isHoverable?: boolean;
+    hasNewThreadedReplies?: boolean;
 };
 
-function ActivityItem({ children, className, isFocused, ...rest }: Props, ref: React.Ref<HTMLLIElement>) {
+function ActivityItem(
+    { children, className, isFocused, isHoverable = false, hasNewThreadedReplies = false, ...rest }: Props,
+    ref: React.Ref<HTMLLIElement>,
+) {
     return (
-        <li className={classNames('bcs-ActivityItem', className, { 'bcs-is-focused': isFocused })} ref={ref} {...rest}>
+        <li
+            className={classNames('bcs-ActivityItem', className, {
```

Comment:
For future reference, may be more readable to pull this out into a separate variable!